build.py

```diff
@@ -32,6 +32,7 @@
 parser.add_argument("--npm", action="store_true", help="Build the npm package")
 parser.add_argument("--play", action="store_true", help="Build the web playground")
 parser.add_argument("--samples", action="store_true", help="Compile the Q# samples")
+parser.add_argument("--notebooks", action="store_true", help="Run Jupyter notebook samples")
```

Comment:
Any thoughts on putting this under the `--integration-tests` step? That step is excluded by default (i.e. if you run `build.py` with no arguments) but runs in CI, as you already know.

(If you do do this, you probably want to update ci.yml as well, to build the pip package in addition to vscode for the integration test run.)

Reply:
I waffled back and forth on this... I do like the idea of them being integration tests since they depend on more components, I just also wanted to make it easy to run them directly without also running the VSCode tests for local testing. Maybe we can make the existing `--integration-tests` a wrapper argument and have two separate arguments for the scenarios, something like `--integration-vscode` and `--integration-notebooks`, where specifying the wrapper one implies enabling all individual integration tests?

Reply:
we talked a bit more about this, capturing where we landed here. Like the `--test` command today, I'll keep it so that `--integration-tests` runs the tests for whatever components have been specified. So for these new ones, you'll have to have included `--pip` and `--widgets` to have built the wheels. Then I'll update the ci.yml to include `--pip` and `--widgets`.